internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,156 @@
+﻿$ErrorActionPreference = 'Stop'; # stop on all errors
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+. $toolsDir\common.ps1
+
+
+$installation_path = "$drive" + "\Program Files\Splunk\OpenTelemetry Collector"
+$program_data_path = "$drive" + "\ProgramData\Splunk\OpenTelemetry Collector"
+$config_path = "$program_data_path\agent_config.yaml"
+
+echo "Checking configuration parameters ..."
+$pp = Get-PackageParameters
+
+$SPLUNK_ACCESS_TOKEN = $pp['SPLUNK_ACCESS_TOKEN']
+$SPLUNK_INGEST_URL = $pp['SPLUNK_INGEST_URL']
+$SPLUNK_API_URL = $pp['SPLUNK_API_URL']
+$install_dir = $pp['install_dir']
+$SPLUNK_HEC_TOKEN = $pp['SPLUNK_HEC_TOKEN']
+$SPLUNK_HEC_URL = $pp['SPLUNK_HEC_URL']
+$SPLUNK_TRACE_URL = $pp['SPLUNK_TRACE_URL']
+$SPLUNK_BUNDLE_DIR = $pp['SPLUNK_BUNDLE_DIR']
+$SPLUNK_REALM = $pp['SPLUNK_REALM']
+
+
+# create config files
+create_program_data
+
+# get param values from config files if they exist
+if (!$SPLUNK_ACCESS_TOKEN) {
+    $SPLUNK_ACCESS_TOKEN = get_value_from_file -path "$program_data_path\token"
+    if (!$SPLUNK_ACCESS_TOKEN) {
+        echo "The 'SPLUNK_ACCESS_TOKEN' parameter was not specified."
+        $SPLUNK_ACCESS_TOKEN = ""
+    } else {
+        echo "Using access token from $program_data_path\token"
+    }
+}
+[System.IO.File]::WriteAllText("$program_data_path\token", "$SPLUNK_ACCESS_TOKEN", [System.Text.Encoding]::ASCII)
+
+# get param values from config files if they exist
+if (!$SPLUNK_REALM) {
+    $SPLUNK_REALM = get_value_from_file -path "$program_data_path\realm"

```

Comment:
This file also has no significance in the collector project.

Reply:
Making the same change as `SPLUNK_ACCESS_TOKEN` ^.